src/pages/home/report/ReportActionsView.js

```diff
@@ -80,14 +85,121 @@ const defaultProps = {
     },
 };
 
-function ReportActionsView(props) {
+/**
+ * Get the currently viewed report ID as number
+ *
+ * @param {Object} route
+ * @param {Object} route.params
+ * @param {String} route.params.reportID
+ * @returns {String}
+ */
+function getReportActionID(route) {
```

Comment:
Remove this function and just do this inline in the component:

```js
const reportActionIDFromRoute = lodashGet(route, 'params.reportActionID', null);
```

Reply:
👍🏻